charts/core-crd/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: core-crd
 description: A Helm chart for Kubernetes
 type: application
-version: 2.4.0
-appVersion: "v2.4.0"
+version: 2.4.1
```

Comment:
@jasmingacic we probably want to change this version in the private repo?

Reply:
I think the private repo already did that? 
https://github.com/calyptia/chart-core-operator/pull/84/files#diff-3066c8293e6aad453968f1fdbd639496206145fdc9808a2bf1a2f4397f4a78ec